crud.go

```diff
@@ -26,6 +26,7 @@ import (
 
 const (
 	SimulatedVBucketCount = 1024 // Used when hashing doc id -> vbno
+	maxDocSize = 20000000 // Used during the write function
```

Comment:
Can we make this a public variable and default to no limit (MaxDocSize=0), and then only apply the limit on write if MaxDocSize > 0?  That would allow SG tests to set as needed.

Should also change to 20x1024x1024.